src/wazuh_modules/wm_syscollector.h

```diff
@@ -18,6 +18,7 @@
 extern const wm_context WM_SYS_CONTEXT;     // Context
 
 #define WM_SYS_LOGTAG ARGV0 ":syscollector" // Tag for log messages
+#define WM_SYSCOLLECTOR_DEFAULT_INTERVAL 3600 // 1 hour
```

Comment:
Functionality nitpick:
```suggestion
#define WM_SYSCOLLECTOR_DEFAULT_INTERVAL W_HOUR_SECONDS
```